frontend/src/metabase/lib/entities.js

```diff
@@ -313,7 +325,11 @@ export function createEntity(def) {
         entityQuery => [...getListStatePath(entityQuery), "fetch"],
       ),
     )((entityQuery = null) => async (dispatch, getState) => {
-      const fetched = await entity.api.list(entityQuery || {});
+      const fetched = await entity.api.list(
+        entityQuery || EMPTY_ENTITY_QUERY,
```

Comment:
because of the default entity query. RTK caches `undefined` and `{}` separately. I think we should at least understand when an empty object was passed here and not by the user and treat it as an empty query.